src/replays/replayitem.py

```diff
@@ -122,7 +122,18 @@ def update(self, message, client):
         
         self.name      = message["name"]
         self.mapname   = message["map"]
-        self.duration  = time.strftime('%H:%M:%S', time.gmtime(message["duration"]))
+        if message['end'] == 4294967295:  # = FFFF FFFF (year 2106) aka still playing
+            seconds = time.time()-message['start']
+            if seconds > 86400:  # more than 24 hours
+                self.duration = str(int(seconds/86400)) + " days<br />&nbsp;! error !"
```

Comment:
If you think something is an error, raise an exception / log it / tell the user with a dialog. Don't do something like this.